tests/test_rocksdb_options.rs

```diff
@@ -16,7 +16,7 @@ mod util;
 
 use std::{fs, io::Read as _};
 
-use rocksdb::{BlockBasedOptions, DataBlockIndexType, Options, ReadOptions, DB};
+use rocksdb::{BlockBasedOptions, DataBlockIndexType, LogLevel, Options, ReadOptions, DB};
```

Comment:
It seems that `LogLevel` is unnecessary import here ?

Reply:
My apologies, it's fixed.